openTRANS/Xml.cs

```diff
@@ -2,54 +2,48 @@
 using System.Text;
 using System.Xml.Serialization;
 
-namespace openTRANS
-{
-    public class XmlCreator
-    {
+namespace openTRANS {
+    public static class Xml {
         //private readonly string xmlVersion = "1.0";
         //private readonly Encoding encoding = new UpperCaseUTF8Encoding(); //Previously Encoding.Utf8; in case of changing also change StringWriter below; would otherwise produce wrong xml encoding attribute
         //private readonly bool standalone = true;
 
         //private readonly XNamespace xsi = "http://www.w3.org/2001/XMLSchema-instance";
 
-        private readonly Order order;
 
-        public XmlCreator(Order order) => this.order = order;
-
-        public string Result
-        {
-            get
-            {
-                //var xdoc = new XDocument(
-                //    new XDeclaration(xmlVersion, encoding.WebName, standalone ? "yes" : "no")
-                //    );
+        static Xml() {
+        }
 
+        public static string Serialize<T>(T o) where T : class, new() {
 
-                var xmlns = new XmlSerializerNamespaces();
-                xmlns.Add(nameof(Common.Namespace.xsi), Common.Namespace.xsi);
-                xmlns.Add(nameof(Common.Namespace.bmecat), Common.Namespace.bmecat);
-                xmlns.Add(nameof(Common.Namespace.xmime), Common.Namespace.xmime);
-                xmlns.Add(nameof(Common.Namespace.xsig), Common.Namespace.xsi);
+            var xmlns = new XmlSerializerNamespaces();
+            xmlns.Add(nameof(Common.Namespace.xsi), Common.Namespace.xsi);
+            xmlns.Add(nameof(Common.Namespace.bmecat), Common.Namespace.bmecat);
+            xmlns.Add(nameof(Common.Namespace.xmime), Common.Namespace.xmime);
+            xmlns.Add(nameof(Common.Namespace.xsig), Common.Namespace.xsi);
 
 
-                XmlSerializer serializer = new XmlSerializer(typeof(Order));
-                using (var writer = new UTF8StringWriter())
-                {
-                    serializer.Serialize(writer, order, xmlns);
-                    return writer.ToString();
-                }
+            XmlSerializer serializer = new XmlSerializer(typeof(o));
```

Comment:
typeof(o) is wrong here..

Use typeof(T)

Reply:
Oh… I see… fixed